docs/source/manual/file_formats/gsb_file_format.rst

```diff
@@ -0,0 +1,103 @@
+.. _file_formats_gsb_xml:
```

Comment:
I guess you forgot to update index.rst
As a result, I cannot preview this file in readthedocs.